x-pack/filebeat/module/aws/cloudtrail/_meta/fields.yml

```diff
@@ -176,6 +176,14 @@
       description: >-
         Identifies the VPC endpoint in which requests were made from a
         VPC to another AWS service, such as Amazon S3.
+    - name: event_category
+      type: keyword
+      description: |-
+        Shows the event category that is used in LookupEvents calls.
+
+         - For management events, the value is management.
```

Comment:
I see `Management` in the generated doc so perhaps the docs should say "the value is Management".